crates/oxc_linter/src/rules/oxc/only_used_in_recursion.rs

```diff
@@ -55,7 +55,8 @@ declare_oxc_lint!(
     /// }
     /// ```
     OnlyUsedInRecursion,
-    correctness
+    correctness,
+    dangerous_fix
```

Comment:
open question -> should this be a suggestion?

Reply:
Looks like dangerous is good 🤔:

```js
let i = 1
function test(arg0) {
    console.log(arguments[0])
    if (i > 0) {
        i--
        return test(arg0);
    }
} 
```